data.sql

```diff
@@ -40,4 +40,33 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
-UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
+
+INSERT INTO public.vets(name, age, date_of_graduation)
+VALUES 
+('William Tatcher', 45, '2000/04/23'),
+('Maisy Smith', 26, '2019/01/17'),
+('Stephanie Mendez', 64, '1981/05/04'),
+('Jack Harkness', 38, '2008/06/08');
+
```

Comment:
Please ensure the specialization data is add to your data.sql file. 👍🏼
![Screenshot_20230503_170034](https://user-images.githubusercontent.com/44624138/235973623-84a2464e-262c-437d-b13f-2dae578eb142.jpg)